tests/test_knowledge_panels.py

```diff
@@ -64,44 +64,20 @@ def test_hunger_game_kp_category_with_country():
     }
 
 
-def test_hunger_game_kp_category_with_value():
-    assert hunger_game_kp(hunger_game_filter="category", value="beer") == {
-        "hunger-game": {
-            "elements": [
-                {
-                    "element_type": "text",
-                    "text_element": {
-                        "html": "<p><a href='https://hunger.openfoodfacts.org/questions?type=category&value_tag=beer'>Answer robotoff questions about beer category</a></p>\n"
-                    },
-                }
-            ]
-        }
-    }
-
-
-def test_hunger_game_kp_brand_with_value():
-    assert hunger_game_kp(hunger_game_filter="brand", value="nestle") == {
-        "hunger-game": {
-            "elements": [
-                {
-                    "element_type": "text",
-                    "text_element": {
-                        "html": "<p><a href='https://hunger.openfoodfacts.org/questions?type=brand&value_tag=nestle'>Answer robotoff questions about nestle brand</a></p>\n"
-                    },
-                }
-            ]
-        }
-    }
-
-
-def test_hunger_game_kp_label_with_value():
-    assert hunger_game_kp(hunger_game_filter="label", value="organic") == {
+def test_hunger_game_kp_category_with_tags():
```

Comment:
why did you remove so many tests ?

- test_hunger_game_kp_category_with_value
- test_hunger_game_kp_brand_with_value
- test_hunger_game_kp_label_with_value

Reply:
Added!